docs/reference/files.md

```diff
@@ -10,6 +10,18 @@ pageClass: page-reference
 > tracked within the `directus_files` system collection. Any requested file transformations are handled on the fly, and
 > are only saved to storage.
 
+## File Security
+
+Data and permissions around files are associated to the `directus_files` collection. 
+
+It is recommended that you only provide public permissions to specific files or file folders (for example, a 'Public' folder), rather than making the whole collection public.
```

Comment:
Maybe we can add something like the following to the end:

> See the user guide for [custom access permissions](https://docs.directus.io/user-guide/user-management/permissions.html#configure-custom-permissions) for more information.

The docs over there have a very helpful video which is awesome for new users.